File: JAVA/Ruster exercise/com/Ruster/Health.java
/**
 *
 * @author JUAN
 */

package com.Ruster;

public class Health {
    private double calculateHealth;

    public Health(double salary) {
        this.calculateHealth = salary * 0.12; // 12%
    }

    public double getCalculateHealth() {
        return this.calculateHealth;
    }
}

File: PHP/Ruster exercise/Ruster/Health.php
<?php
class Health extends Person {
    private float $calculateHealth;

    public function __construct(float $salary) {
        $this->salary = $salary;
        $this->calculateHealth = $salary * 0.12;
    }

    public function getCalculateHealth(): float {
        return $this->calculateHealth;
    }

    public function getDescription(): string {}
    public function getNetSalary(): float {}
}


